wallet/wallet.h

```diff
@@ -249,13 +249,18 @@ struct wallet_payment {
 	struct list_node list;
 	u64 id;
 	u32 timestamp;
+
+	/* The combination of these two fields is unique: */
 	struct sha256 payment_hash;
+	u64 parallel_id;
```

Comment:
So the MPP acronym is for "multi-parallel payment", and not like I thought "multi-part payment"? :wink:

Reply:
Yeah, naming predated the MPP name.  I like 'partid'.